### ISSUE TITLE
Inline source maps with .es6 file extension

### ISSUE THREAD

**Issue by jfaissolle** [node-inspector/node-inspector#580]
It seems that when the filename extension is not .es6, inline source maps are not taken into account. I have two files:
test.js
```javascript
'use strict';
require('babel/register');
var test = require('./test2');
test();
```
and 
test2.es6
```javascript
'use strict';

module.exports = () => {
  let name = 'world';
  console.log(`Hello ${name}`);
};
```
`test2.es6` produces an inline source map but it is not used by `node-inspector`. If I rename it to `test2.js`, the source map works as expected.

**Comment by bajtos** [comment]
I am afraid I don't have enough time to investigate this issue in detail,I suspect the problem may be related to what I have found in coffee-script transpiler - see https://github.com/node-inspector/node-inspector/issues/224#issuecomment-38785891. Here is the behaviour I would expect:

 - The transpiled javascript source has the source map embedded via data url (per rfc2397, see also strong-data-uri). No test2.map file is created.
 - The transpiled source is executed, the script file name is set to test2.js (must be a different file name!)
- The source map points to test2.es6 as the original source.

Possibly related: https://github.com/node-inspector/node-inspector/issues/100#issuecomment-31475540 and https://github.com/babel/babel/issues/306

**Comment by 3y3** [comment]
Fixed in 0.10.2.
Feel free to reopen this issue, if I'm wrong here.
